packages/kbn-search-api-panels/index.tsx

```diff
@@ -37,50 +37,57 @@ export const WelcomeBanner: React.FC<WelcomeBannerProps> = ({
   image,
   showDescription = true,
 }) => (
-  <EuiFlexGroup justifyContent="spaceBetween">
-    <EuiFlexItem grow={false}>
-      {/* Reversing column direction here so screenreaders keep h1 as the first element */}
-      <EuiFlexGroup justifyContent="flexStart" direction="columnReverse" gutterSize="s">
-        <EuiFlexItem grow={false}>
-          <EuiTitle className="serverlessSearchHeaderTitle" size="s">
-            <h1>
-              {i18n.translate('searchApiPanels.welcomeBanner.header.title', {
-                defaultMessage: 'Get started with Elasticsearch',
-              })}
-            </h1>
-          </EuiTitle>
-        </EuiFlexItem>
-        {Boolean(user) && (
+  <>
+    <EuiSpacer size="xxl" />
+    <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+      <EuiFlexItem grow={2}>
+        {/* Reversing column direction here so screenreaders keep h1 as the first element */}
+        <EuiFlexGroup justifyContent="flexStart" direction="columnReverse" gutterSize="s">
           <EuiFlexItem grow={false}>
-            <EuiTitle size="xxxs">
-              <h2>
-                {user
-                  ? i18n.translate('searchApiPanels.welcomeBanner.header.greeting.customTitle', {
-                      defaultMessage: 'Hi {name}!',
-                      values: { name: user.full_name || user.username },
-                    })
-                  : i18n.translate('searchApiPanels.welcomeBanner.header.greeting.defaultTitle', {
-                      defaultMessage: 'Hi!',
-                    })}
-              </h2>
+            <EuiTitle className="serverlessSearchHeaderTitle" size="l">
```

Comment:
I know you didn't add this, but checking I don't think this `serverlessSearchHeaderTitle` css class exists anywhere anymore. We should be able to remove this.